lib/src/browser/disposable_browser.dart

```diff
@@ -133,16 +134,33 @@ class _InnerDisposable extends disposable_common.Disposable {
 /// composition to include the [Disposable] machinery without changing
 /// the public interface of our class or polluting its lifecycle.
 class Disposable implements disposable_common.Disposable {
+  /// The name of the factory function added to the window that produces
+  /// [LeakFlag] objects when called.
```

Comment:
"when called (with no args)"?

Reply:
Oh, it takes one arg. I'd include that in the description here because this seems like the docs most consumers will be reading.